cucumber/authenticators_azure/features/authn_azure_hosts.feature

```diff
@@ -117,8 +141,23 @@ Feature: Azure Authenticator - Different Hosts can authenticate with Azure authe
     Errors::Authentication::AuthnAzure::InvalidApplicationIdentity
     """
 
+  Scenario: Host with incorrect system-assigned-identity annotation is authorized
```

Comment:
this scenario is denied/unauthorized

Reply:
need to change here too..(you changed for user-assigned-identity only ...

Reply:
oh yeah missed it. thanks!